addons/medical_status/stringtable.xml

```diff
@@ -71,9 +72,11 @@
             </Key>
             <Key ID="STR_ACE_Medical_Status_ivFlowRate_DisplayName">
                 <English>IV Transfusion Flow Rate</English>
+                <German>Bluttransfusions Fließrate</German>
```

Comment:
Plasma and Saline are not blood, but are also IV's.
Just "Transfusions"

Reply:
Ok